docs/questions/query-builder/expressions/datetimeadd.md

```diff
@@ -44,26 +44,11 @@ You can use the [`between`](../expressions-list.md#between) or [`interval`](../e
 | Boolean                 | ❌                   |
 | JSON                    | ❌                   |
 
-This table uses `timestamp` and `datetime` interchangeably---just make sure that your dates and times aren't stored as string or a number data types in your database.
+This table uses `timestamp` and `datetime` interchangeably---just make sure that your dates and times aren't stored as strings or numbers in your database.
 
 ## Limitations
 
-You can use `datetimeAdd` to _calculate_ relative dates given a column of date values, but unfortunately Metabase doesn't currently let you _generate_ a relative date (such as today's date).
-
-For example, if you want to check if today's date falls between **Opened On** and **Finish By** in the [Coffee example](#calculating-an-end-date):
-
-- Ask your database admin if there's table in your database that stores dates for reporting (sometimes called a date dimension table).
-- Create a new question using the date dimension table, with a filter for "Today".
-- Turn the "Today" question into a model.
-- Create a left join between **Coffee** and the "Today" model on `[Opened On] <= [Today]` and `[Finish By] >= [Today]`.
-
-The result should give you a **Today** column that's non-empty if today's date falls inside the coffee freshness window:
-
-| Coffee                 | Opened On  | Finish By  | Today      | 
-|------------------------|------------|------------|------------|
-| DAK Honey Dude         | 2022-10-31 | 2022-11-14 | 2022-11-10 |
-| NO6 Full City Espresso | 2022-11-07 | 2022-11-21 | 2022-11-10 |
-| Ghost Roaster Giakanja | 2022-11-27 | 2022-12-11 |            |
+If you're using MongoDB, `datetimeAdd` will only work on versions 5 and up.
```

Comment:
Worth keeping [an eye on this conversation](https://metaboat.slack.com/archives/C03SHMZ266R/p1669212168286559) to confirm driver compatibility.